demos/multi_channel/face_detection/CMakeLists.txt

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set(TARGET_NAME "multi-channel-face-detection-demo")
+set(TARGET_NAME "multi_channel_face_detection")
```

Comment:
I think you should add `_demo` to each of the demo subfolders and executable names, to preserve the convention that each demo ends with `_demo`.

Reply:
Not each: `object_detection_demo_faster_rcnn`, `object_detection_demo_ssd_async`, `object_detection_demo_yolov3_async`, `action_recognition`, `multi_camera_multi_person_tracking`, `object_detection_demo_ssd_async`, `object_detection_demo_yolov3_async`.

So I thought should it be either multi_channel`_demo`_face_detection or multi_channel_face_detection`_demo`. But ended up with complete removing of the word since  the demos are already in `demos` folder.

Reply:
The various object detection demos do at least have `_demo` in their name. As for `action_recognition` and `multi_camera_multi_person_tracker`, I would say that they're in the minority, so they're wrong. :-)

> But ended up with complete removing of the word since the demos are already in demos folder.

I can sympathize with this argument, but it doesn't justify breaking the convention. We should be moving towards more uniformity, rather than less. You could change the convention by renaming _all_ demos, but that belongs in another PR (and IMO, not worth doing at all).

Reply:
added